lib/services/awsAdapter.js

```diff
@@ -118,6 +119,24 @@ function AWSAdapter (options) {
 	this.beginPolling = function (result) {
 		var self = this;
 		var instanceIPAddress;
+
+		var objectSchema = Joi.object().keys({
```

Comment:
I think this is a bit of duplication. Instead of creating a new schema to test for validity, we could just instantiate a new `Instance`, which will do that for us. We can then pass the `Instance`'s id to the `SshPolling` function.